lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2684,12 +2740,15 @@ StringRef getTFCompatibleFuncName(SILFunction *fn) {
   return fnName;
 }
 
-/// Lower the specified SIL function (which was formed by the partitioner)
-/// into a TensorFlow graph, and encode into a vector of bytes.
-///
-std::vector<char> tf::lowerTFGraph(
+// `graphFnNameForCaller` is for caller.
+// If `isAcceleratorOnly`, graphFnName is set to the primaryGraphFnName for a TF
+// graph node to call; otherwise, it is set to entryFnBaseName, for the host
+// runtime to call.
+static std::vector<char> lowerTFGraphHelper(
```

Comment:
Please add a comment explaining what this function does.  Also, please pick a name better than "helper" :-), please describe what this does.

Reply:
Done.